include/eve/module/real/core/function/regular/simd/x86/frac.hpp

```diff
@@ -35,12 +35,12 @@ namespace eve::detail
   // -----------------------------------------------------------------------------------------------
   // 256 bits implementation for xop
   template<floating_real_scalar_value T, typename N>
-  EVE_FORCEINLINE auto frac_(EVE_SUPPORTS(avx_),
-                             wide<T, N, x86_256_> const &a0) noexcept
+  EVE_FORCEINLINE wide<T, N> frac_(EVE_SUPPORTS(avx_), wide<T, N> const &a0) noexcept
```

Comment:
why the change from `auto`?

Reply:
Because `_mm256_frcz_pd` does not return `wide`